packages/substrate_metadata/lib/old/types.dart

```diff
@@ -0,0 +1,55 @@
+class OldTypes {
```

Comment:
@leonardocustodio  just brainstorming names, not all of them make sense
```suggestion
class LegacyTypes {
```
```suggestion
class UnusedTypes {
```

```suggestion
class UnsupportedTypes {
```

```suggestion
class ThrowbackTypes {
```

Reply:
@pastre `LegacyTypes` sounds good to me.